tests/e2e-net-test.sh

```diff
@@ -7,15 +7,14 @@
 error_info() {
     echo -n "INFO: "
     echo $@
-    exit 1
+    exit 0
```

Comment:
This function is called error_info, so don't you want to return a value != 0

Reply:
it should be called exit_info, fixing

Reply:
fixed.